src/main/java/io/debezium/connector/vitess/connection/VStreamOutputMessageDecoder.java

```diff
@@ -335,15 +342,37 @@ private void handleFieldMessage(Binlogdata.VEvent vEvent) {
     }
 
     private Table resolveTable(String schemaName, String tableName, List<ColumnMetaData> columns) {
+        List<String> pkColumnNames = new ArrayList<>();
+        String uniqueColumnName = null;
```

Comment:
Let's maybe rename it to uniqueKeyColumnName ?

Reply:
fixed